datashuttle/tui/app.py

```diff
@@ -64,12 +131,35 @@ def on_directory_tree_directory_selected(
         with directory name. Double-click time is set to the
         Windows default (500 ms).
         """
-
         click_time = monotonic()
         if click_time - self.prev_click_time < 0.5:
-            self.query_one("#subject").value = str(event.path)
+            if event.path.stem.startswith("sub-"):
+                self.query_one("#subject").value = str(event.path.stem)
+            if event.path.stem.startswith("ses-"):
+                self.query_one("#session").value = str(event.path.stem)
+                self.query_one("#subject").value = str(event.path.parent.stem)
```

Comment:
Could re-write this docstring slightly to incorporate the newer changes (i.e. it will change subject or session depending on the prefix).

Reply:
Good catch, thanks Joe! I've changed the docstring now!